.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: ci
```

Comment:
I'm sorry but this does not work.

I don't know why but the dingo execution fails, on other hand the previous travis didn't work neither.